website_multi_theme/views/ir_ui_view.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). -->
+<odoo>
+
+    <record model="ir.ui.view" id="view_view_form_extend">
+        <field name="model">ir.ui.view</field>
+        <field name="inherit_id" ref="base.view_view_form"/>
+        <field name="arch" type="xml">
+
+            <xpath expr="//notebook" position="inside">
+                <page string="Multi-website" attrs="{'invisible': [('copy_ids', '=', [])]}">
+                    <separator string="Copies"/>
+                    <field name="copy_ids"/>
+                </page>
+                <page string="Multi-website" attrs="{'invisible': [('origin_view_id', '=', False)]}">
```

Comment:
2 pages called the same?

Better 1 page and 2 groups.

Remember to add a `name="multiwebsite"`, to enable possible inheritance.

Reply:
The idea is don't pass attributes twice in each node here:

                    <separator string="Copies"/>
                    <field name="copy_ids"/>

Reply:
I can add different names, if it's the concern

    name="multiwebsite_origin_view"
    name="multiwebsite_copied_view"